configure.ac

```diff
@@ -851,6 +851,19 @@ if test x"$with_libjxl" != x"no"; then
   )
 fi
 
+if test x"$with_libjxl" = x"yes"; then
+  save_LIBS="$LIBS"
+  LIBS="$LIBS $LIBJXL_LIBS"
+  # Introduced in 0.6
+  AC_CHECK_FUNCS(JxlEncoderInitBasicInfo,[
+     AC_DEFINE(HAVE_LIBJXL_JXLENCODERINITBASICINFO,1,
+               [define if you have heif_image_handle_get_raw_color_profile.])
```

Comment:
```suggestion
               [define if you have JxlEncoderInitBasicInfo.])
```

Reply:
Heh yes I noticed after merge, sigh. I'll fix it.